docs/modules/ROOT/pages/capacity-planning.adoc

```diff
@@ -255,41 +264,47 @@ Don't rely just on CPU usage when benchmarking your cluster. Simulate
 production workload and measure the throughput and latency instead. The
 task manager of Hazelcast can be configured to use the CPU aggressively.
 As an example, see https://hazelcast.com/blog/idle-green-threads-in-jet/[this benchmark]: the CPU usage was close to 20% with just 1000 events/s. At 1m items/s
-the CPU usage was 100% even though Jet still could push around 5m
+the CPU usage was 100% even though Jet still could push around 5 million
 items/s on that machine.
 
 === Network
 
 Hazelcast uses the network internally to shuffle data and to replicate the
-back-ups. Network is also used to read input data from and to write
+backups. The network is also used to read input data from and to write
 results to remote systems or to do RPC calls when enriching. In fact a
-lot of Hazelcast jobs are network-bound. Using a 10 Gbit or faster network
+lot of Hazelcast jobs are network-bound. A 1Gbit network connection is a 
```

Comment:
... A 1 Gbit network ...

(I think the convention is put a space between the value and unit)

Reply:
(Yep, makes sense.) Changed.